app/src/main/java/com/codingblocks/cbonlineapp/Modules.kt

```diff
@@ -14,7 +15,7 @@ val viewModelModule = module {
     viewModel { HomeViewModel(get(), get(), get(), get()) }
     viewModel { MyCourseViewModel(get(), get(), get(), get(), get()) }
     viewModel { NotificationViewModel(get()) }
-
+    viewModel { LeaderboardViewModel(get()) }
```

Comment:
Here get means satisfying the parameters which have been defined the view model by automatically one of the methods defined in this class see HomeViewModel if you still don't get it